README.md

```diff
@@ -1,5 +1,4 @@
-**WARNING:If you are on Ubuntu or an Ubuntu-based distribution you'll need to do `sudo apt install libx11-dev` before using Pine-jinx!**
-
+**WARNING:If you are on Ubuntu or an Ubuntu-based distribution you'll need to do `sudo apt install libx11-dev` before using Pine-jinx! Also please note that this fork is a heavy WIP and only grabs 1.1.4 as of now. This only affects the master build, not the LDN build (as its hosted on patreon)**
```

Comment:
This also seems outdated

Reply:
Yeah I'll change that back. I only made it when I began work on it last night.